documentation/yarn.lock

```diff
@@ -0,0 +1,10369 @@
+# This file is generated by running "yarn install" inside your project.
```

Comment:
Do we need this file here in this repo? I think we removed it before because it wasn't necessary and we get all the dependabot PRs then

Reply:
Actually I think we can ignore the yarn.lock file. Good point. I will keep that in mind while testing

Reply:
We decided we wanted to make sure that the yarn.lock files stays. You need to make sure to maintain the plugins you use as we now also import them into the wiki